app/views/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if @user.errors.any? %>
+  <div id="error_explanation">
+    <div class="alert alert-danger">
+    <%= t("signup.c2") %>
+    <%= pluralize(@user.errors.count, t("signup.error")) %>.
```

Comment:
nghiên cứu view số nhiều số ít bằng I18n https://viblo.asia/p/i18n-in-rails-part-2-GrLZDAO3lk0

Reply:
oke anh